timeutil/schedule_test.go

```diff
@@ -538,6 +538,35 @@ func (ts *timeutilSuite) TestScheduleNext(c *C) {
 			// expecting February, 27th, 10:00, 20 days and 23 hours
 			// from now
 			next: "503h-503h",
+		}, {
+			// from last Monday of the month to the Tuesday of
```

Comment:
From last Monday or from first Monday? The schedule below says `mon1` so I assume it's the first Monday of the month.